js/adapt-stateful-session.js

```diff
@@ -118,6 +119,49 @@ define([
 			this.setupEventListeners();
 		},
 
+		/*
+		* SCORM 1.2 requires that the identifiers in cmi.interactions.n.student_response for choice and matching activities be a character from [0-9a-z].
+		* When numeric identifiers are used this function attempts to map identifiers 10 to 35 to [a-z]. Resolves issues/1376.
+		*/
+		_checkResponse: function(response, responseType) {
+			if (!response) return response;
+			if (responseType != 'choice' && responseType != 'matching') return response;
+
+			var zero = '0',
+				nine = '9',
+				a = 'a',
+				z = 'z';
```

Comment:
Is it necessary to actually declare variables for these?

Couldn't they be referenced inline as the strings '0', '9', etc.?

Also, not sure about the single var pattern going on here.

Reply:
Resolved